File: C/14-4-2026/bai1.c
#include <stdio.h>

int m, n, x;
int main () {
    printf("Nhập số hàng và cột: ");
    scanf("%d %d", &m, &n);
    int a[m][n];
    for (int i = 0; i < m; i++)
    {
        printf("Nhập giá trị theo hàng a[%d]: ", i);
        scanf("%d", &a[i]);
    }
    for (int j = 0; j < n; j++) {
            printf("Nhập giá trị theo cột a[%d]: ", j);
            scanf("%d", &a[j]);
        }
    printf("Nhập số cần tìm: "); scanf("%d",&x);
    for (int i = 0; i < m; i++) {
        int exist;
        for (int j = 0; j < n; j++) {
            if (a[i][j] == x)
            {
                exist = 1;
            } else {exist = 0;}
        }
        if (exist == 1) {
            printf("%d tồn tại trong mảng\n", x);
        } else {printf("%d không tồn tại trong mảng\n", x);}
    }
}

File: Python/thihsg_2/ôn tập tổng hợp/buổi 2/bai1.py
import math
r=int(input('Nhập bán kính: '))
x0,y0=map(int,input('Nhập vị trí tâm O theo x,y: ').split())
a,b=map(int,input('Nhập vị trí điểm A bất kì theo x,y: ').split())
d=math.sqrt((a-x0)**2+(b-y0)**2)
if d==r:
    print('Điểm A nằm trong hình tròn')
else:
    print('Điểm A nằm ngoài đường tròn')